packages/react-components/lib/robots/tests/robot-accordion.test.tsx

```diff
@@ -0,0 +1,18 @@
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+import { render } from '@testing-library/react';
+import React from 'react';
+import { RobotAccordion } from '..';
+
+const robot: RomiCore.RobotState = {
+  name: 'test',
+  battery_percent: 1,
+  location: { level_name: 'test_level', x: 0, y: 0, yaw: 0, t: { sec: 0, nanosec: 0 } },
+  mode: { mode: RomiCore.RobotMode.MODE_PAUSED },
+  model: 'test_model',
+  task_id: 'test_task_id',
+  path: [],
+};
+
+test('smoke test', () => {
```

Comment:
maybe we can add a snapshot test too

Reply:
I did do snapshot testing at first, but material-ui is not designed with snapshot testing in mind and they do not recommend it. You would need to go through some hacks to make the components stable under snapshots, I think it's probably not worth the trouble.